cmstestsuite/unit_tests/grading/scoretypes/GroupMinTest.py

```diff
@@ -131,31 +134,51 @@ def test_max_scores_number(self):
 
     def test_compute_score(self):
         s1, s2, s3 = 10.5, 30.5, 59
-        parameters = [[s1, "1_*"], [s2, "2_*"], [s3, "3_*"]]
+        parameters = [[0, "0_*"], [s1, "1_*"], [s2, "2_*"], [s3, "3_*"]]
         gmin = GroupMin(parameters, self._public_testcases)
         sr = self.get_submission_result(self._public_testcases)
 
         # All correct.
         self.assertComputeScore(gmin.compute_score(sr),
```

Comment:
nit: weird indentation here. maybe put the `gmin.compute_score(sr),` in the next line.

Similarly for other tests as well

Reply:
Done!